code/modules/mob/living/basic/space_fauna/netherworld/blankbody.dm

```diff
@@ -31,35 +31,12 @@
 
 /mob/living/basic/blankbody/Initialize(mapload)
 	. = ..()
-	var/datum/callback/health_changes_callback = CALLBACK(src, PROC_REF(health_check))
 	AddElement(/datum/element/swabable, CELL_LINE_TABLE_NETHER, CELL_VIRUS_TABLE_GENERIC_MOB, 1, 0)
-	AddComponent(/datum/component/damage_buffs, health_changes_callback)
-
-/mob/living/basic/blankbody/proc/health_check(mob/living/attacker)
-	if(health < maxHealth * 0.25)
-		health_low_behaviour()
-	else if (health < maxHealth * 0.5)
-		health_medium_behaviour()
-	else if (health < maxHealth * 0.75)
-		health_high_behaviour()
-	else
-		health_full_behaviour()
-
-/mob/living/basic/blankbody/proc/health_full_behaviour()
-	melee_damage_lower = 2
-	melee_damage_upper = 6
-
-/mob/living/basic/blankbody/proc/health_high_behaviour()
-	melee_damage_lower = 4
-	melee_damage_upper = 8
-
-/mob/living/basic/blankbody/proc/health_medium_behaviour()
-	melee_damage_lower = 8
-	melee_damage_upper = 12
-
-/mob/living/basic/blankbody/proc/health_low_behaviour()
-	melee_damage_lower = 10
-	melee_damage_upper = 20
+	AddComponent(
```

Comment:
Can't these all fit on one line? seems like it could work on github, is it just fugly in the IDE?

Reply:
I think I multilined it before realising I didn't need to provide as many properties as I was